cmd/juju/commands/synctools.go

```diff
@@ -170,13 +171,14 @@ type syncToolsAPIAdapter struct {
 
 func (s syncToolsAPIAdapter) FindTools(majorVersion int, stream string) (coretools.List, error) {
 	result, err := s.syncToolsAPI.FindTools(majorVersion, -1, "", "", "")
+	if params.IsCodeNotFound(result.Error) || errors.IsNotFound(err) {
```

Comment:
cmd code should not be concerned with params errors, this should be handled in the client api code